ci/build-unix.yml

```diff
@@ -22,6 +22,7 @@ steps:
 
   - bash: ./fmt.sh --test
     displayName: 'Platform-agnostic lints and checks'
+    condition: and(succeeded(), eq(variables['Agent.OS'], 'Linux'))
```

Comment:
I'm not going to ask why `eq(variables['Agent.OS'], 'Linux')` alone is not sufficient.

Reply:
That will still run it if the step before fails afaik.

Reply:
Yes: the default implicit condition if you don't specify one is `succeeded()`, but if you do specify one it replaces that.